v2_papers/2016/ITC_flowStatistics.json

```diff
@@ -0,0 +1,307 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet traffic classification based on flows' statistical properties with machine learning ",
+    "authors": [
+      "Alina Vlăduţu",
+      " Dragoş Comăneci",
+      "Ciprian Dobre"
+    ],
+    "publication_name": "Management of the Internet of things and big data",
+    "publication_type": "peer_reviewed_journal",
+    "year": 2016,
+    "organization_publishers": [
+      "wiley"
+    ],
+    "pages_number_of": 14,
+    "bibtex": {
+      "type": "",
+      "volume": "",
+      "issue": "",
+      "pages": ""
```

Comment:
(same as previous papers)
These bibtex fields must not be empty strings. If there's no content for them, remove them.
"type" field is mandatory though.
For the bibtex fields I recommend copying the values from the bibtex provided by google scholar (google for the paper in scholar, and look at the citation button).